lib/hets-rabbitmq-wrapper.rb

```diff
@@ -7,18 +7,42 @@ module HetsRabbitMQWrapper
   def self.run
     connection = Bunny.new
     connection.start
+
+    min_parsing_version(connection)
+  end
+
+  def self.instance_version
+    #get version from HetsInstance
+  end
+
+  def self.min_parsing_version(connection)
     channel = connection.create_channel
-    # worker will only accept one message at time
-    channel.prefetch(1)
-    request_queue =
-      channel.queue('hets-request', durable: true, auto_delete: false)
-    result_queue = channel.queue('hets-result')
+    channel.exchange_declare('ex_min_parsing_version', 'x-recent-history',
+                             {'x-recent-history-length' => 1})
+    q_min_parsing_version = channel.queue('q_min_parsing_version',
+                                          durable: true, auto_delete: false)
+    q_min_parsing_version.bind('ex_min_parsing_version')
+    q_min_parsing_version.
+      subscribe(block: false,
+                timeout: 0) do |delivery_info, properties, version|
+      subscribe(version, connection)
+    end
+  end
 
-    request_queue.subscribe(manual_ack: true,
-                            block: true,
-                            timeout: 0) do |delivery_info, _properties, _body|
-      # TODO: push body to hets
-      channel.ack(delivery_info.delivery_tag)
+  def self.subscribe(version, connection)
```

Comment:
This is because of long lines which I had to break. I guess, its okay in this case.